MinecraftClient/Resources/lang/zh-CHS.ini

```diff
@@ -355,147 +361,147 @@ cmd.useitem.use=使用了一个物品
 
 # AutoAttack
 bot.autoAttack.mode=未知的攻击模式：{0}，使用单一模式作为默认值。
-bot.autoAttack.priority=未知优先级：{0}，使用距离优先级作为默认值。
+bot.autoAttack.priority=未知优先模式：{0}，使用距离优先作为默认值。
 bot.autoAttack.invalidcooldown=攻击冷却时间值不能小于0，已使用自动作为默认值
 
 # AutoCraft
-bot.autoCraft.cmd=自动创建ChatBot命令
-bot.autoCraft.alias=自动创建ChatBot命令别名
+bot.autoCraft.cmd=自动制作ChatBot命令 # TOCHECK
```

Comment:
One thing I want to confirm before finalizing:  
In similar strings of other bots, the name of the bot is included here, but 'Auto-crafting' is not the name of this bot. I'm not sure if I should translate 'Auto-crafting' or leave it as is.

Reply:
`Auto-crafting` mean automatically craft something. It is not the name of the bot so you may translate it into Chinese.